src/graph/InterimResult.cpp

```diff
@@ -377,5 +377,20 @@ std::unique_ptr<InterimResult> InterimResult::getInterim(
 
     return std::make_unique<InterimResult>(std::move(rsWriter));
 }
+
+Status InterimResult::applyTo(std::function<Status(const RowReader *reader)> visitor,
+                              int64_t limit) const {
```

Comment:
What‘s the use of `limit`?

Reply:
We can apply the callback to interim result with a limit.